packages/list/ListEditing.js

```diff
@@ -78,6 +60,7 @@ ListEditing.Prototype = function() {
           path: path,
           startOffset: 0
         });
+				args.selection = selection;
```

Comment:
Also need to fix the space issue. There seem to be 4 tabs here instead of 4 spaces or 1 tab.